ml-libs/CMakeLists.txt

```
@@ -114,6 +114,7 @@ if(THEROCK_ENABLE_MIOPEN)
       -DMIOPEN_USE_MLIR=OFF # TODO: enable
       -DMIOPEN_TEST_DISCRETE=OFF
       -DBoost_COMPILER=${_THEROCK_MIOPEN_OVERRIDE_BOOST_COMPILER}
+      "-DMIOPEN_INSTALL_GPU_DATABASES=\"${THEROCK_AMDGPU_TARGETS}\""
```

Are these extra escaped quotes necessary? I didn't need them when I tested myself using `-DTHEROCK_AMDGPU_FAMILIES=gfx110X-all`, which mapped to `MIOPEN_INSTALL_GPU_DATABASES:STRING=gfx1100;gfx1101;gfx1102;gfx1103`
```suggestion
      "-DMIOPEN_INSTALL_GPU_DATABASES=${THEROCK_AMDGPU_TARGETS}"
```

Note that we also set some other variables, in case you want to reuse one:
```
//From super-project
AMDGPU_TARGETS:STRING=gfx1100;gfx1101;gfx1102;gfx1103

//GPU targets to compile for
GPU_BUILD_TARGETS:STRING=gfx1100;gfx1101;gfx1102;gfx1103

//From super-project
GPU_TARGETS:STRING=gfx1100;gfx1101;gfx1102;gfx1103

//List of AMDGPU targets for which to install databases
MIOPEN_INSTALL_GPU_DATABASES:STRING=gfx1100;gfx1101;gfx1102;gfx1103
```